crypto/modes/asm/ghashv8-armx.pl

```diff
@@ -18,10 +18,11 @@
 #
 # June 2014
 #
-# Initial version was developed in tight cooperation with Ard
-# Biesheuvel <ard.biesheuvel@linaro.org> from bits-n-pieces from
-# other assembly modules. Just like aesv8-armx.pl this module
-# supports both AArch32 and AArch64 execution modes.
+# Initial version was developed in tight cooperation with Ard Biesheuvel
+# from bits-n-pieces from other assembly modules. Just like aesv8-armx.pl
+# this module supports both AArch32 and AArch64 execution modes.
+# Initial version was developed in tight cooperation with Ard Biesheuvel
+# of Linaro.
```

Comment:
It says "Initial version was developed" two times...

Reply:
fixed.